packages/ui/Card/Card.js

```diff
@@ -0,0 +1,215 @@
+/**
+ * Unstyled card components and behaviors to be customized by a theme or application.
+ *
+ * @module ui/Card
+ * @exports Card
+ * @exports CardBase
+ * @exports CardDecorator
+ */
+
+import EnactPropTypes from '@enact/core/internal/prop-types';
+import kind from '@enact/core/kind';
+import PropTypes from 'prop-types';
+import compose from 'ramda/src/compose';
+
+import ComponentOverride from '../ComponentOverride';
+import ForwardRef from '../ForwardRef';
+import Image from '../Image';
+import {Cell, Column, Row} from '../Layout';
+
+import componentCss from './Card.module.less';
+
+function ImageOverride ({imageComponent, ...rest}) {
+	return ComponentOverride({
+		component: imageComponent,
+		...rest
+	});
+}
+
+/**
+ * A basic card without any behavior.
+ *
+ * @class CardBase
+ * @memberof ui/Card
+ * @ui
+ * @public
+ */
+const CardBase = kind({
+	name: 'ui:Card',
+
+	propTypes: /** @lends ui/Card.CardBase.prototype */ {
+		/**
+		 * The caption node to be displayed with the image.
+
+		 * @type {Node}
+		 * @public
+		 */
+		children: PropTypes.node,
+
+		/**
+		 * Called with a reference to the root component.
+		 *
+		 * When using {@link ui/Card.Card}, the `ref` prop is forwarded to this component
+		 * as `componentRef`
+		 *
+		 * @type {Object|Function}
+		 * @public
+		 */
+		componentRef: EnactPropTypes.ref,
+
+		/**
+		 * Customizes the component by mapping the supplied collection of CSS class names to the
+		 * corresponding internal elements and states of this component.
+		 *
+		 * The following classes are supported.
+		 *
+		 * @type {Object}
+		 * @public
+		 */
+		css: PropTypes.object,
+
+		/**
+		 * The component used to render the image component.
+		 *
+		 * @type {Component|Element}
+		 * @default {@link ui/Image.Image}
+		 * @public
+		 */
+		imageComponent: EnactPropTypes.componentOverride,
+
+		/**
+		 * The layout orientation of the component.
+		 *
+		 * @type {('horizontal'|'vertical')}
+		 * @default 'vertical'
+		 * @public
+		 */
+		orientation: PropTypes.oneOf(['horizontal', 'vertical']),
+
+		/**
+		 * A placeholder image to be displayed before the image is loaded.
+		 *
+		 * @type {String}
+		 * @public
+		 */
+		placeholder: PropTypes.string,
+
+		/**
+		 * Applies a selected visual effect to the image.
+		 *
+		 * @type {Boolean}
+		 * @default false
+		 * @public
+		 */
+		selected: PropTypes.bool,
+
+		/**
+		 * String value or object of values used to determine which image will appear on a specific
+		 * screenSize.
+		 *
+		 * @type {String|Object}
+		 * @public
+		 */
+		src: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
+
+		/**
+		 * Determines whether the text will be placed over the image.
+		 *
+		 * @type {Boolean}
+		 * @default false
+		 * @public
+		 */
+		textOverlay: PropTypes.bool
```

Comment:
Let's rename it to `captionOverlay` to be more general.

Reply:
It's fixed.